resources/main.mjs

```diff
@@ -269,6 +277,23 @@ class MainBenchmarkClient {
         this._showSection("about", true);
     }
 
+    _getFormattedJSONResult()
+    {
+        const indent = "    ";
+        return JSON.stringify(this._measuredValuesList, undefined, indent);
+    }
+
+    copyJsonResults()
+    {
+
+        navigator.clipboard.writeText(this._getFormattedJSONResult());
```

Comment:
This API doesn't exist in Safari or Firefox.

Reply:
It does seem to work both in Firefox 109.0.1 (64-bit) and Safari Version 16.3 on my mac.